### DISCUSSION TITLE
Kube-bench roadmap 0.7.0

### DISCUSSION THREAD

**Discussion by yoavrotems** [aquasecurity/kube-bench#919] [Development]
**Hello all** 🥳 
We want to update and improve kube-bench! 
For the next kube-bench version I would like to hear from you what do you think of kube-bench and what new features would you like to see 😸 

I started by opening two issues 
[Add new statuses](#916) and [New flag! --status](#918)

I will remind all that we are going to deprecate some commands

> 
> ### Deprecate master and node commands
> 
> Commands:
> `kube-bench master` 
> and 
> `kube-bench node`
> will be daprecated in next version and currently will print warnning about it. 
> You should run instead: 
> `kube-bench run --targets="master,node"`
> 
> 

We consider to deprecate also Pgsql output, and Junit output since it doesn't seems to be used, would like to hear if someone here using it. 

Feel free to add suggestions or jump in on issues and enhancements you can contribute to

**Comment by steve-heslouin** [comment]
Sending result to GCP security command center would be really amazing :)

**Comment by julien-sugg** [comment]
Hi,

Junit deprecation may have some impact for GitLab users that want to integrate the kube-bench reports within their pipelines and MR status as GitLab only support Junit xml format. (See https://docs.gitlab.com/ee/ci/unit_test_reports.html#how-it-works)

Additional Context/Remark:

~I just started using kube-bench today for my own integrations needs and it appears that the Junit report is broken (at least for v0.6.3). Long story short, the generated report is not compliant with the junit xml format and is missing the `xml meta declaration element` and `testsuites root element`. (See https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd#L18)~

Edit: just found https://github.com/aquasecurity/kube-bench/issues/883 related to the remark.

**Comment by yoavrotems** [comment]
Hey @julien-sugg thanks for your inputs, I will try to see if we can tell how much is this in use with kube-bench users, because if we are talking on just small amount of people we will need to discuss about it and decide if deprecating it or not

**Comment by samkulkarni20** [comment]
Hi @yoavrotems, do you know when we can expect next release of kube-bench? Either v0.6.6 or v0.7.0? There are some changes to AKS-1.0 scan config in master branch, which my team at Rancher is interested in. Any pointers will be helpful.

**Reply by yoavrotems** [reply]
In the next few days we will release v0.6.6 version! 🥳 @samkulkarni20

**Reply by yoavrotems** [reply]
https://github.com/aquasecurity/kube-bench/discussions/1080 😺  @samkulkarni20
